src/components/pages/ImageClassification/ImageClassificationTable/ImageClassificationObservationTable.js

```
@@ -0,0 +1,626 @@
+                                rowSpan={numSubRows + (totalUnknown > 0 ? 1 : 0)}
+                                data-tooltip={file.original_image_name}
+                                onClick={() => handleImageClick(file)}
+                                cursor={
+                                  file.classification_status.status === 3 ? 'pointer' : 'default'
+                                }
+                              >
+                                <ImageWrapper>
+                                  <Thumbnail imageUrl={file.thumbnail} />
+                                </ImageWrapper>
+                              </TdWithHoverText>
+                            </>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)